### DISCUSSION TITLE
Tutorial Example shows results which are inconsistent with the code

### DISCUSSION THREAD

**Discussion by leon-stier** [gatsbyjs/gatsby#36735] [Help]
The 5th part of the Gatsby tutorial series (_Part 5: Transform Data to Use MDX_) instructs the reader to query the excerpt for the article and render it in a paragraph. Since the excerpt doesn't include the markup for the list it should show up as a single paragraph; and it does. The tutorial however shows a screenshot of the site rendering the excerpt as a list. 
This doesn't align with the provided code which becomes apparent when looking at the linked [finished version](https://gatsbytutorialexample.gatsbyjs.io/blog), which actually renders the excerpt as a single paragraph and not as a list.

This has send me into a frustrating search for errors in my code while the result is actually not achievable like that.

_This is what the tutorial claims should be the result_
![image](https://user-images.githubusercontent.com/33227102/193692104-e1e4d5f5-bd70-4463-9872-5c4ce3caf90f.png)
_And this is what the hosted finished example looks like_
![image](https://user-images.githubusercontent.com/33227102/193692023-754bb8c2-5eb5-48eb-8424-4c8d557191f6.png)

**Comment by LekoArts** [comment]
Hi!

We recently revamped the `gatsby-plugin-mdx` and with this also how the excerpt is created. We use https://github.com/syntax-tree/hast-util-excerpt for this. It's purpose is to be used in e.g. meta tags so I think how it handles lists or other things right now is "kinda" working as expected. Anyhow, we'll need to update the tutorial to display the correct image, thanks for pointing this out. If you want you can put up a PR yourself :)
